identity/src/main/java/com/android/identity/DataTransportBlePeripheralServerMode.java

```diff
@@ -51,6 +51,7 @@ public class DataTransportBlePeripheralServerMode extends DataTransportBle {
     UUID mCharacteristicClient2ServerUuid = UUID.fromString("00000002-a123-48ce-896b-4c76973373e6");
     UUID mCharacteristicServer2ClientUuid = UUID.fromString("00000003-a123-48ce-896b-4c76973373e6");
     // Note: Ident UUID not used in peripheral server mode
+    UUID mCharacteristicL2CAPUuid =         UUID.fromString("0000000a-a123-48ce-896b-4c76973373e6");
```

Comment:
Same as with the other file, it wasn't clear why this is the right UUID and I had to do a bunch of investigative work to convince myself this is correct! Suggest to add the following comment:

```
In _mdoc peripheral server mode_ the _mdoc_ acts as the GATT server and the _mdoc reader_ acts as the
GATT client. According to ISO 18013-5 Table A.1 this means that in _mdoc peripheral server mode_
the GATT server (for the _mdoc_) should advertise
UUID 0000000A-A123-48CE896B-4C76973373E6 and the GATT client (for the _mdoc reader_) should
connect to that UUID.
```

and also rename the variable name to `mCharacteristicL2CAPUuidMdoc`.